zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -56,6 +55,8 @@
 import org.apache.zeppelin.user.AuthenticationInfo;
 import org.apache.zeppelin.user.Credentials;
 
+import static org.apache.commons.lang.StringUtils.*;
```

Comment:
As per [project styleguide#s3.3.1-wildcard-imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) - could you please replace whiledcard imports?